packages/account/src/Sections/Profile/PersonalDetails/__tests__/validation.spec.tsx

```diff
@@ -58,32 +62,86 @@ describe('getPersonalDetailsValidationSchema', () => {
         const isValid = await validationSchema.isValid(non_eu_valid_data);
         expect(isValid).toBe(false);
     });
+
+    it('should return empty object for virtual account', () => {
+        const validationSchema = getPersonalDetailsValidationSchema(false, true);
+        expect(validationSchema.fields).toEqual({});
+    });
 });
 
+const account_settings = {
+    first_name: 'John',
+    last_name: 'Doe',
+    place_of_birth: 'US',
+    address_state: 'state_test',
+};
+const mock_residence_list = [
+    { value: 'UK', text: 'United Kingdom' },
+    { value: 'US', text: 'United States' },
```

Comment:
Why do we need to check for these 2 countries? we are not supporting any clients from both of these countries.

Reply:
They are mock Data. I can replace them with names the app uses